src/modules/python.rs

```diff
@@ -26,19 +26,19 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
         return None;
     }
 
-    let use_pyenv = context
+    let pyenv_version_name = context
```

Comment:
You should be able to do the following once the module has been created to make things slightly simpler.

```rust
let pyenv_version_name = module
        .config_value_bool("pyenv_version_name")
        .unwrap_or(false);
```

Reply:
I think my idea was not making the module until `select_python_version` returns `Some`, I don't know if we want to keep that behavior or not

Reply:
I think it's best if we perform that check after the module is created. That way we can skip the check if the module is disabled (which is checked during module creation).